packages/filefeeds/src/config.ts

```diff
@@ -91,8 +91,46 @@ export interface FileFeedsParams {
   saveSession?: boolean
 
   /**
-   * A session token to resume an existing session. 
+   * A session token to resume an existing session.
    * Automatically populated by the SDK if `saveSession` is true.
    */
   resumeToken?: string
```

Comment:
we can move `resumeToken` to just be in launch params then too?

Reply:
I think it should be a non-export part of the `launch` behavior—meaning, taking it completely out of the Params types. Let's do that in a separate PR, though.